test/addons-napi/test_error/test_error.cc

```diff
@@ -15,9 +15,64 @@ napi_value checkError(napi_env env, napi_callback_info info) {
   return result;
 }
 
+napi_value throwExistingError(napi_env env, napi_callback_info info) {
+  napi_value message;
+  napi_value error;
+  NAPI_CALL(env, napi_create_string_utf8(env, "existing error", -1, &message));
+  NAPI_CALL(env, napi_create_error(env, message, &error));
+  NAPI_CALL(env, napi_throw(env, error));
+  return nullptr;
```

Comment:
Just noting for the future: it would be excellent if these n-api thrown errors followed the same conventions as the `internal/errors` module. That is, include a `code` property and the ` [{code}]:` in the name.